nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/settings/ui/parameter-providers/edit-parameter-provider/edit-parameter-provider.component.scss

```diff
@@ -17,6 +17,10 @@
 
 @use '@angular/material' as mat;
 
+h2.mdc-dialog__title::before {
+    height: 0;
+}
```

Comment:
Why do we need this for the edit dialogs but not for any of the others? Is it just because we have divs in the h2?